packages/xrpl/test/models/XChainAccountCreateCommit.test.ts

```diff
@@ -0,0 +1,161 @@
+import { assert } from 'chai'
+
+import { validate, ValidationError } from '../../src'
+import { validateXChainAccountCreateCommit } from '../../src/models/transactions/XChainAccountCreateCommit'
+
+/**
+ * XChainAccountCreateCommit Transaction Verification Testing.
+ *
+ * Providing runtime verification testing for each specific transaction type.
+ */
+describe('XChainAccountCreateCommit', function () {
+  let tx
+
+  beforeEach(function () {
+    tx = {
+      Account: 'rHb9CJAWyB4rj91VRWn96DkukG4bwdtyTh',
+      XChainBridge: {
```

Comment:
nit:  Should there be a test with issued currencies?

Reply:
I think that's a bit overkill for a model test - I believe I have one in the integration tests in #2301. IIRC this specific transaction only takes XRP.